### PR TITLE
Update PushOver notification count

### PR DESCRIPTION
**Pull request by daudfatihah**
#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.

#### Description of Change
change to count message using message_row class quantity instead app_counter(not exist)

### CODE REVIEW

**Comment by mcmxcdev** [comment]
Hey @daudfatihah, thanks a lot for the PR!

Please go into the `package.json` of the pushover recipe and change the version from v1.1.0 to v1.1.1

Only then users will be able to receive the updated code from this PR, and we will be able to merge this.

File: recipes/pushover/webview.js [hunk 1]
```diff
@@ -6,10 +6,8 @@ const _path = _interopRequireDefault(require('path'));
 
 module.exports = Ferdium => {
   const getMessages = () => {
-    const totalMessages = document.querySelector(
-      '#app_0 > div.app_counter',
-    ).innerHTML;
-    Ferdium.setBadge(totalMessages);
+    const totalMessages = document.getElementsByClassName("message_row").length;
+    Ferdium.setBadge(totalMessages)
```
Comment (mcmxcdev) [inline recipes/pushover/webview.js:10]: This line is missing a semicolon which will probably break the prettier formatting check:

```suggestion
    Ferdium.setBadge(totalMessages);
```

**Comment by daudfatihah** [comment]
> Hey @daudfatihah, thanks a lot for the PR!
> 
> Please go into the `package.json` of the pushover recipe and change the version from v1.1.0 to v1.1.1
> 
> Only then users will be able to receive the updated code from this PR, and we will be able to merge this.

Done update to v1.1.1

**Review by vraravam — CHANGES_REQUESTED** [review]
please update this PR with the version bump in the `package.json` for this recipe

**Review by vraravam — APPROVED** [review]

**Review by vraravam — CHANGES_REQUESTED** [review]
@daudfatihah - the PR is failing ; probably because of lint issues. Please make sure to follow the contribution guide and run all the steps before this PR can be accepted and merged

**Comment by vraravam** [comment]
@daudfatihah - CI build is broken. Please fix by going through the commands detailed in the contribution guide. Please do not request for review without fixing those.

**Review by mcmxcdev — APPROVED** [review]

**Review by SpecialAro — APPROVED** [review]
LGTM! Haven't tested because I don't use PushOver

**Comment by SpecialAro** [comment]
@all-contributors please add daudfatihah for code

**Pull request merged**